### PR TITLE
Better calendar

### PR DESCRIPTION
**Pull request by MichaelVargas31**
Merging Act 2, take 1

### CODE REVIEW

File: Mytinerary/View Controllers/DailyCalendarViewController.m [hunk 1]
```diff
@@ -11,19 +11,26 @@
...
- 
  
     Maybe NSCalendar could be useful?
 
  
  Questions:
     - Should formatter be defined as a property of the Daily View Controller?
+    - Possible to add addEventWith() to scheduledEventView.m file?? outsourcing?
+    - Currently iterating through 'events' array of pointers in itinerary, using the ID to fetch said
+            event from parse. More efficient way?
```
Comment (fireph) [inline Mytinerary/View Controllers/DailyCalendarViewController.m:22]: there must be a way to fetch multiple IDs in 1 fetch with parse... 🤔

Comment (fireph) [reply Mytinerary/View Controllers/DailyCalendarViewController.m:22]: Maybe this will help:

https://stackoverflow.com/a/31432281

**Comment by fireph** [comment]
lots of commented code in this PR, any particular reason for that? Is it going to be used later?

File: Mytinerary/View Controllers/DailyCalendarViewController.m (hunk 1 above)
Comment (MichaelVargas31) [reply Mytinerary/View Controllers/DailyCalendarViewController.m:22]: Thats super helpful!

**Pull request merged**